### DISCUSSION TITLE
remove one side from ring

### DISCUSSION THREAD

**Discussion by apithy-isidro-martinez** [tailwindlabs/tailwindcss#3421] [Help]
is there a way that allow set or remove only one side of the ring? like border does, i want to remove the top ring

**Comment by tomsdob** [comment]
Could you please elaborate the problem a bit more or post a code snippet/picture of what you're trying to achieve?

**Comment by andreaselia** [comment]
I'm afraid not, this isn't something CSS natively supports.
